api/python/provisioner/inputs.py

```diff
@@ -1112,7 +1112,27 @@ def pillar_value(self):
 
             return res
         elif self.is_remote():
-            return self.source
+            return {
+                f'os': {
+                    'source': (f'{self.source}/os'),
```

Comment:
parenthesis are not needed, please remove

Reply:
Done